spec/support/active_record/schema.rb

```diff
@@ -40,6 +40,10 @@
     t.column :created_at, :datetime
     t.column :updated_at, :datetime
     t.column :favourite_device, :string
+
+    if ENV['DB'] == 'POSTGRES'
+      t.column :favourite_colours, :json
```

Comment:
Is there a way to test this on other DBs as well? perhaps using serialization to save the array as a string column?

Reply:
I considered this.

However, if the field is not :json, then active record does not automatically cast, and audited_changes would contains strings.

We'd get ```audited_changes: ["['blue', 'red']", "['blue']"]```
instead of ```[[blue, red],[blue]]```

Therefore, it's just another string, and already covered in other tests.

If you disagree I'd be happy to come up with something a bit more explicit

Reply:
@tbrisker thoughts?

Reply:
sure, this is better in any case. I think we could use serialize somehow to test other dbs, but i won't block on that.

Reply:
I haven't looked into it too deeply, but at least in Rails 5 this can be tested using `audit.revision`. That will raise a `SerializationTypeMismatch` when trying to get the revision for a `create` audit that contains an array serialized using Rails' `serialize` (because it gets a string rather than an array).